### DISCUSSION TITLE
How do I never open nvim-tree on startup?

### DISCUSSION THREAD

**Discussion by jakkan** [nvim-tree/nvim-tree.lua#2472] [Q&A] [answered]
The wiki contains information about how to open nvim-tree on startup, but how do I tell nvim-tree to never open at startup? E.g. when I run "nvim ./some/folder" I don't want nvim-tree to open.

Thanks for taking your time

**Comment by alex-courtis** [comment, marked as answer]
`:help nvim-tree.hijack_directories.enable` tells nvim-tree to open when a directory is opened.

Set that to false.
